src/query/api/v1/handler/prometheus/remote/tag_values.go

```diff
@@ -135,9 +132,8 @@ func (h *TagValuesHandler) parseTagValuesToQuery(
 		FilterNameTags:   [][]byte{nameBytes},
 		TagMatchers: models.Matchers{
 			models.Matcher{
-				Type:  models.MatchRegexp,
-				Name:  nameBytes,
-				Value: matchValues,
+				Type: models.MatchField,
+				Name: nameBytes,
```

Comment:
Perhaps remove `var matchValues = []byte(".*")` ? Looks like it's not used anymore.